recipes/evv4esm/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "evv4esm" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 905d71e860feda96e6107b2075a502ccc7e8d7425f60d567c9b44d97cb129c50
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - evv = evv4esm.__main__:main
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - six
+    - numpy
+    - scipy
+    - pandas
+    - livvkit
+    - netcdf4
+    - matplotlib
```

`matplotlib-base` should include everything you need, the `matplotlib` package only is the additional Qt dependency which is probably not needed.
```suggestion
    - matplotlib-base
```